whipper/command/cd.py

```diff
@@ -341,7 +350,19 @@ def doCommand(self):
         else:
             logger.info("creating output directory %s", dirname)
             os.makedirs(dirname)
-
+        self.coverArtPath = None
+        if (self.options.fetch_cover_art in {"embed", "complete"} and
+                importlib.util.find_spec("PIL") is None):
+            logger.warning("the cover art option '%s' won't be honored "
+                           "because the 'pillow' module isn't available",
+                           self.options.fetch_cover_art)
+        elif self.options.fetch_cover_art in {"file", "embed", "complete"}:
+            self.coverArtPath = self.program.getCoverArt(
+                                    dirname,
+                                    self.options.fetch_cover_art,
+                                    self.program.metadata.mbid)
+        if self.options.fetch_cover_art == "file":
+            self.coverArtPath = None  # This way the image won't be embedded
```

Comment:
This seems like a hacky way to handle this. Good deal with the comment though and no immediate suggestion for another way. It can be improved later.

Reply:
Yeah, it is a bit hacky indeed but I think the logic isn't too much convoluted.

Reply:
Updated comment with code tag.